USAGE.md

```diff
@@ -2710,6 +2710,9 @@ This endpoint has a helper, check it out [here](https://github.com/sendgrid/send
     throw ex;
   }
   ```
+
+If a SENDGRID_CHECK_TEMPLATES environment variable is set to "true", a template_id in the body of the email will be checked against the templates in the user's account; an exception will be thrown if no template exists with the specified template_id.
+
```

Comment:
I'd rather not have this be set via env vars, and instead have a separate helper method that can be explicitly called by the user as needed. I agree with these comments about being against pre-emptive API calls:
https://github.com/sendgrid/sendgrid-nodejs/issues/778#issuecomment-434724311
https://github.com/sendgrid/sendgrid-python/issues/645#issuecomment-429547814